<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRegister</class>
 <widget class="QWidget" name="addRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加寄存器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>起始地址</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_startAddr">
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Hex</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MySpinBox" name="spinBox_num">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
